timemachine/fe/rbfe.py

```diff
@@ -431,38 +435,46 @@ def estimate_relative_free_energy(
 
     temperature = DEFAULT_TEMP
     initial_states = setup_initial_states(single_topology, host_config, temperature, lambda_schedule, seed)
-    protocol = SimulationProtocol(n_frames=n_frames, burn_in=10000, steps_per_frame=1000)
+    protocol = SimulationProtocol(n_frames=n_frames, n_eq_steps=n_eq_steps, steps_per_frame=1000)
 
     if keep_idxs is None:
         keep_idxs = [0, -1]  # keep first and last frames
     assert len(keep_idxs) <= len(lambda_schedule)
     combined_prefix = get_mol_name(mol_a) + "_" + get_mol_name(mol_b) + "_" + prefix
 
-    try:
-
-        return estimate_free_energy_given_initial_states(
-            initial_states, protocol, temperature, combined_prefix, keep_idxs
-        )
-
-    except Exception as old_exc:
-
-        raise SimulationException(initial_states, protocol, combined_prefix) from old_exc
+    return estimate_free_energy_given_initial_states(initial_states, protocol, temperature, combined_prefix, keep_idxs)
```

Comment:
Was this change intentional? Seems like we still want the `SimulationException` to be able to attach additional info?

Reply:
Updated to store the pickle on exception in rbfe